src/components/Navbar.jsx

```diff
@@ -0,0 +1,13 @@
+import '../index.css'
+
+export default function Navbar(){
+    return(
+        <nav>
+            <ul className="nav--list">
+               <li>Drop-down</li> 
```

Comment:
Questions:
- Is 'Drop-down' going to be a hamburger menu? What will that contain? 
- Why is 'Logo' part of the navigation? I know it should be in the 'Header' component, but is it a navigation link?
- Cart should be a button element.

Maybe this code is just acting as a placeholder. I guess I just don't see the items in this component as navigation elements. Also wasn't @ken-ellingsen working on the Header component? If so, he should make sure to reconcile any changes he's making with yours.

Reply:
We had also talked about building mobile-first. This looks styled for desktop - which again, might just be placeholder.

Reply:
Thanks for the feedback-- I wanted to get a little more boilerplate on the page so I went ahead and threw some elements in. I tried to make it based off the mobile design, but it really doesn't look anything like it now that you mention it. The goal was to use this to help test routing

Reply:
Can we close this pull request @dylanmullarkey? That way we don't have any merge conflict when Ken pushes up the Header component...